RELEASES.md

```diff
@@ -0,0 +1,14 @@
+# Releasing Create React Native App
+
+This document contains checklists to be done before releasing new versions.
+
+## Verify .flowconfig is up to date after updating expo-sdk
+
+After upgrading the [expo-sdk](https://github.com/exponent/exponent-sdk) version (which transitively updates the `react-native` version), ensure that the [.flowconfig](react-native-scripts/template/.flowconfig) template is up to date.
+
+Easiest way to do this, is:
+1. Use [this handy chart](https://docs.expo.io/versions/latest/sdk/index.html#sdk-version) to find out the underlying `react-native` versions of the old and new `expo-sdk`.
```

Comment:
Probably want to point contributors to the CRNA one https://github.com/react-community/create-react-native-app/blob/master/VERSIONS.md